pkg/topolvm-scheduler/cmd/root.go

```diff
@@ -83,6 +109,6 @@ func Execute() {
 }
 
 func init() {
-	rootCmd.Flags().StringVar(&config.listenAddr, "listen", ":8000", "listen address")
-	rootCmd.Flags().Float64Var(&config.divisor, "divisor", 1, "capacity divisor")
+	rootCmd.Flags().StringVar(&config.ListenAddr, "listen", ":8000", "listen address")
```

Comment:
Why do you keep this as a command-line flag?
It is now read from config YAML, ain't it?